src/main/java/CollectingCharactersDataViaTerminal.java

```diff
@@ -0,0 +1,12 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class CollectingCharactersDataViaTerminal {
+    List<String> charactersNamesList = new ArrayList<>();
+    Scanner namesInput = new Scanner(System.in);
+
+    List<Integer> charactersWeightsList = new ArrayList<>();
```

Comment:
Trzymając w osobnych listach nazwę postaci i jej wagę, nie jesteś w stanie być pewien, która waga tyczy się, jakiej postaci. Możesz tylko zgadywać na podstawie tego indexsu tych list. Natomiast to się rozwali jeżeli zdecydujemy się w jakiś sposób posortować listę postaci czy też wagi, dane te nie są powiązane ze sobą w kodzie.

Proponuję stworzyć klasę, która powiąże ze sobą te dwie wartości. 
Czyli klasę, która ma pole z nazwą postaci i z wagą dla tej nazwy postaci. 
W sumie ta klasa już tutaj istnieje i nazywa się `Character` https://github.com/RandomUsername559/IsaacCharacterGeneratorApplication/blob/master/src/main/java/Character.java